macros/utils/cross_db_utils/get_profile_creation_query.sql

```diff
@@ -17,8 +17,26 @@ GRANT SELECT ON FUTURE VIEWS IN SCHEMA {{ parameters["database"] }}.{{ parameter
 {% endmacro %}
 
 
+{% macro redshift__get_profile_creation_query(parameters) %}
+-- Create redshift user with unrestricted access to query history (allows Elementary to see queries generated by
+-- any user)
+CREATE USER {{ parameters["user"] }} WITH PASSWORD '{{ parameters["password"] }}' SYSLOG ACCESS UNRESTRICTED;
+
+-- Grant read access to the Elementary schema
+GRANT USAGE ON SCHEMA {{ parameters["schema"] }} TO {{ parameters["user"] }};
+GRANT SELECT ON ALL TABLES IN SCHEMA {{ parameters["schema"] }} TO {{ parameters["user"] }};
+ALTER DEFAULT PRIVILEGES IN SCHEMA {{ parameters["schema"] }} GRANT SELECT ON TABLES TO {{ parameters["user"] }};
+
+-- Grant metadata access to tables in the warehouse
+GRANT SELECT ON svv_table_info to {{ parameters["user"] }};
```

Comment:
This table is actually one of the few system tables in Redshift where you have to explicitly give access (for others you can always select and you can only see what you get access to).

It looks like it has interesting information so I added it (even though it's not related to query history)